src/roman_datamodels/testing/factories.py

```diff
@@ -539,6 +539,30 @@ def create_gain_ref(**kwargs):
 
     return stnode.GainRef(raw)
 
+
+def create_ipc_ref(**kwargs):
+    """
+    Create a dummy IpcRef instance with valid values for attributes
+    required by the schema.
+
+    Parameters
+    ----------
+    **kwargs
+        Additional or overridden attributes.
+
+    Returns
+    -------
+    roman_datamodels.stnode.IpcRef
+    """
+    raw = {
+        "data": _random_array_float32((32, 32)),
```

Comment:
I don't think it really matters, but we really expect the IPC array to be 3x3, have all positive values, and sum to one.  A slightly more realistic fake IPC kernel would be:
kernel = np.abs(_random_array_float32((3, 3)))
kernel /= np.sum(kernel)